.eslintrc

```diff
@@ -6,7 +6,6 @@
     "eqeqeq": 2,
     "guard-for-in": 2,
     "indent": [2, 2, {"SwitchCase": 1}],
-    "keyword-spacing": 2,
```

Comment:
As far as I can see, this option is not part of eslint:recommended.
Shouldn't we keep it?